app/assets/stylesheets/application.scss

```diff
@@ -44,3 +44,23 @@ $govuk-typography-use-rem: false;
 .release__view-diff {
   float: right;
 }
+
+.non-production-version {
+  @include govuk-font(16);
+  margin: 0 15px 15px 15px;
+  border: 1px solid #B01117;
+  background-color: govuk-colour('red');
+  color: govuk-colour('white');
+  padding: 1em;
```

Comment:
please change to use the govuk spacing instead of hard coded numbers for both your padding and your margin:
margin: govuk-spacing(3);
margin-top: 0;

padding: govuk-spacing(3);